content/release-10-18-0/platform-10-18-0.md

```diff
@@ -159,6 +134,30 @@ Build version</th>
 <td>UI</td>
 </tr>
 
+<tr>
+<td>Authentication</td>
+<td>In the single sign-on configuration page, users can now enable and configure the external token validation.</td>

```

Comment:
is it "single-sign-on" or "single sign-on"?

Reply:
It is "single sign-on"